### ISSUE TITLE
5 Everyday Foods That Hurt Male Fertility

### ISSUE THREAD

**Issue by sophieH33** [sophieH33/health#1]
Most men tend to worry about the sperm count they are producing while trying to conceive with their partners, and sometimes they don’t take into consideration their sperm quality. This is causing about fifteen percent of couple are facing the problem of infertility. 

There are numerous factors that cause male infertility such as the blockage, sperm production, and  motility Some people face issues due to use of alcohol and tobacco. However, some other factors like environmental and behavioral factors could affect your ability to get pregnant. 

For example, foods a man eats will also affect your fertility. Men should avoid these five foods that hurt your fertility:
1.	High mercury fish
Both males and females are affected by the mercury content in some of the fishes they eat. The mercury damages the omega-3 in the fishes. This will harm the omega-3 absorption in your body and also decrease your fertility. King Mackerel, shark, tuna, tilefish, swordfishes and more have the highest content of mercury and should be avoided at all costs if trying to get married. 

Avoid shellfishes too as they can overloaded with zinc.
2.	Full fat dairy
Even though milk is crucial to many people for numerous health benefits. Full fat dairy products like milk, cheese, heavy creams, yogurts and more even in as less as two servings will lead to the decrease in the sperm’s mobility. 
3.	Processed meats
Men love meat, everyone loves meat but eating the kind not right for you can compromise your fertility. Researchers suggest that men who eat a lot of processed meat like burger patties, bacon, salami, hot dogs, bacon, and other packed or frozen meats in large amounts can lead to twenty three percent lesser sperm. It is probably because the meats when processed might have residues of hormonal content that can affect your reproductive system.
4.	Soda
Studies show that frequent consumption of sugary drinks and beverages in larger amounts are strongly connected to the motility of the sperm. If you drink these beverages such as soda, sweet immune boosting drinks, sports drinks and teas, your insulin resistance can raise and will lead to the oxidative stress which will damage your sperm.
5.	Caffeine and alcohol
Research has concluded that extra consumption of coffee, tea, alcohol, energy drinks and other beverages can significantly decrease the sperm count which will subsequently will affect your fertility. If you are a coffee addict, or regular drinker alongside a poor diet, you can be in for a very long term damage. Copious amounts of alcohol and caffeine will cause permanent oxidative stress and cause infertility.  

In Conclusion:
Remember infertility is as big a problem for males as it is for females. Over 30 percent of fertility issues are associated to men. This is why you need to eat foods that will raise your chances of getting pregnant when you do start trying with your partner.

[Modafinil online](http://www.healthkartrx.com/modafinil-online-200mg.php)
